addons/l10n_ch/views/qr_invoice_wizard_view.xml

```diff
@@ -5,8 +5,6 @@
         <field name="model">l10n_ch.qr_invoice.wizard</field>
         <field name="arch" type="xml">
             <form string="QR printing encountered a problem">
-                <field name='nb_qr_inv' invisible="1"/>
```

Comment:
Add a comment so that it passes the CI in stable, and remove in master to avoid breaking potential inheritance based on these fields